aws-frauddetector-detector/src/aws_frauddetector_detector/helpers/api_helpers.py

```diff
@@ -565,6 +565,12 @@ def call_get_external_models(frauddetector_client, model_endpoint: str = None):
     return frauddetector_client.get_external_models(**args)
 
 
+@api_call_with_debug_logs
+def call_get_model_version(frauddetector_client, model_id: str, model_type: str, model_version_number: str):
```

Comment:
why was `@retry_not_found_exceptions` removed from this call?

Reply:
Provided model version should already exist before stack creation, so we don't have any eventual consistency cases we need to retry